src/ILLink.Tasks/build/Microsoft.NET.ILLink.targets

```diff
@@ -205,6 +205,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Warning Condition="'$(TrimmerDefaultAction)' != '' And $([MSBuild]::VersionGreaterThan('$(TargetFrameworkVersion)', '6.0'))"
              Text="Property 'TrimmerDefaultAction' is deprecated in .NET 7 and will be ignored. Use TrimMode instead." />
 
+    <Warning Condition="'$(TrimMode)' == 'copyused' And $([MSBuild]::VersionGreaterThan('$(TargetFrameworkVersion)', '6.0'))"
+             Text="TrimMode 'copyused' is deprecated in .NET 7. Use TrimMode 'full' or 'partial' instead." />
```

Comment:
The only thing which worries me about this is that it's basically impossible to suppress this warning. Which might break people's builds.